applications/callflow/src/cf_util.erl

```diff
@@ -959,6 +960,18 @@ account_timezone(Call) ->
             whapps_config:get(<<"accounts">>, <<"timezone">>, ?DEFAULT_TIMEZONE)
     end.
 
+-spec maybe_use_variable(wh_json:object(), whapps_call:call()) -> api_binary().
+maybe_use_variable(Data, Call) ->
```

Comment:
Same as in `cf_pivot` but I have no idea what this function is meant for. Docs describing what you're attempting to do here would be helpful.